docs/data/product/dea-fractional-cover-landsat/_access.md

```diff
@@ -2,6 +2,11 @@
 
 % ## Use constraints
 
+:::{dropdown} Which folder contains the Landsat 5/7 and Landsat 8/9 data on NCI and AWS?
+
+The data for Landsat 5 and Landsat 7 is in the `2-5-0` folder whereas the data for Landsat 8 and Landsat 9 is in the `2-5-1` folder on both NCI and AWS. This is because these two sets of data are produced using different versions of the product.
```

Comment:
@tebadi I've written this explanation to prevent confusion. Could confirm if what I've written is correct?

Reply:
the version update or discrepancy is NOT due to the input is different, but the different parameters used in `2.5.1` in ls8/9 than `2.5.01

Reply:
Thank you. I've changed the wording to:

`This is because these two sets of data are produced using different algorithm parameters.`

Is it correct to call them 'algorithm parameters'.

Reply:
more of extra parameters. LS8/9 have extra set of parameters.

Reply:
Thanks. That's ok, I don't need to be too specific. I can just say 'different algorithm parameters'.